driver/driver.go

```diff
@@ -106,6 +112,7 @@ func NewDriver(ep, token, url string) (*Driver, error) {
 	})
 
 	return &Driver{
+		doVolTag: os.Getenv(doVolTagEnv),
```

Comment:
We shouldn't assign it here. Instead this should be passed to `cmd/do-csi-plugin` via a flag, such as `--do-tag`. And then in our controller we would use the still the same environment variable in this form:

```
        - name: csi-do-plugin
          image: digitalocean/do-csi-plugin:v1.0.0
          args :
            - "--endpoint=$(CSI_ENDPOINT)"
            - "--token=$(DIGITALOCEAN_ACCESS_TOKEN)"
            - "--url=$(DIGITALOCEAN_API_URL)"
            - "--do-tag=$DIGITALOCEAN_TAG_VOLUMES)"
```

This is to prevent any side effects on the binary and `cmd/do-csi-plugin` should the only place we pass in external values)

Reply:
Done